tasks.py

```diff
@@ -218,3 +218,15 @@ def integration(context):
         "docker-compose up --build --force-recreate --quiet-pull --exit-code-from integration integration",
         env={"PYTHON_VER": context["nautobot_ansible"]["python_ver"]},
     )
+
+
+@task
+def galaxy_build(context):
+    """Build the collection."""
+    context.run("ansible-galaxy collection build .")
+
+
+@task
+def galaxy_build_force(context):
```

Comment:
I don't think this needs a separate task. Add the force argument to the `galaxy_build` task instead.

Reply:
I will fix this up.

Reply:
Suggestion below